pyproject.toml

```diff
@@ -8,8 +8,10 @@ readme = "README.md"
 packages = []
 
 [tool.poetry.dependencies]
-python = "3.9.14"
+python = "3.9.15"
```

Comment:
We need to update the [Dockerfile](https://github.com/mitodl/mitxpro/blob/master/Dockerfile#L1) as well. Local build throws an error of Python version mismatch.